packages/coinstac-pipeline/src/pipeline-manager.js

```diff
@@ -48,12 +53,23 @@ module.exports = {
     let io;
     let socket;
     const remoteClients = {};
-    const missedCache = {};
+    // TODO: const missedCache = {};
 
     const waitingOnForRun = (runId) => {
       const waiters = [];
       activePipelines[runId].clients.forEach((client) => {
-        if (remoteClients[client][runId] && !remoteClients[client][runId].currentOutput) {
+        if ((remoteClients[client][runId]
```

Comment:
putting this `remoteClients[client][runId]` into a variable would improve readability here and also probably make the code smaller.

Reply:
👍